sklearn/utils/tests/test_extmath.py

```diff
@@ -648,6 +650,8 @@ def test_softmax():
 
 
 def test_stable_cumsum():
+    if np_version < (1, 19):
```

Comment:
Argh. Making so many errors. Should be in bed. Should take a break from this stuff, too!